src/hooks/useGetCourses.ts

```diff
@@ -30,8 +30,8 @@ export const useGetCourses = (info: StudentWithoutChapel) => {
   });
 
   return {
-    majorRequired: results[0],
-    majorElective: results[1],
-    generalRequired: results[2],
+    MAJOR_REQUIRED: results[0],
+    MAJOR_ELECTIVE: results[1],
+    GENERAL_REQUIRED: results[2],
   };
```

Comment:
combine 옵션을 사용하고 반환할 때 data.result 배열이 비어있는지 여부도 같이 반환해주면 어떨까요?

```ts
const getArrayState = <T>(array: T[]): 'FILLED' | 'EMPTY' =>
  array.length > 0 ? 'FILLED' : 'EMPTY';
```

```suggestion  
combine: (results) => ({
      MAJOR_REQUIRED: {
        ...results[0],
        arrayState: getArrayState(results[0].data?.result ?? []),
      },
      MAJOR_ELECTIVE: {
        ...results[1],
        arrayState: getArrayState(results[1].data?.result ?? []),
      },
      GENERAL_REQUIRED: {
        ...results[2],
        arrayState: getArrayState(results[2].data?.result ?? []),
      },
    }),
```